### PR TITLE
Increases idle time from 15 to 40 min:

### PR DESCRIPTION
**Pull request by btylerburton**
WHY: this allows DOI json time to download

# Pull Request

Related to [LINK TO ISSUE]

## PR TASKS

- [ ] The actual code changes.
- [ ] Tests written and passed.
- [ ] Any changes to docs?
- [ ] Any dependent PRs needed?

### CODE REVIEW

**Comment by btylerburton** [comment]
Also made a few recommended syntax modifications, let me know if those look ok as well...

File: bin/check-and-renew [hunk 1]
```diff
@@ -93,9 +93,9 @@ function check_running_job {
 
       # Check the age of the last log
       # If there are only two lines, there are no logs
-      if [[ $ok_to_restart == "YES" && $((`log_count` > 2)) == '1' ]]; then
+      if [[ $ok_to_restart == "YES" && $(($(log_count) > 2)) == '1' ]]; then
         # if the timestamp is younger than 15 mins from start of script
```
Comment (FuhuXia) [inline bin/check-and-renew]:
```suggestion
        # if the timestamp is younger than 40 mins from start of script
```

**Review by nickumia-reisys — DISMISSED** [review]
Should be safe enough 🍇

**Review by FuhuXia — CHANGES_REQUESTED** [review]

File: bin/check-and-renew (hunk 1 above)
Comment (nickumia-reisys) [reply bin/check-and-renew]: Oh... is this a long running change?

Comment (FuhuXia) [reply bin/check-and-renew]: some data.json take longer to download.
https://gsa-tts.slack.com/archives/C2N85536E/p1692638742516909

**Comment by btylerburton** [comment]
@FuhuXia i incorporated your change to the doc string. still says you have changes though i can't see any more.

**Review by FuhuXia — APPROVED** [review]

**Pull request merged**